docs/agile-with-CEA/roles-and-responsibilities.rst

```diff
@@ -0,0 +1,110 @@
+
+This is a guide of roles and responsibilities in CEA. We use this to more formally coordinate our work.
+To check who is currently holding each position, we invite you to check `here
+<https://cityenergyanalyst.com/people/>`_ for more information.
+
+.. image:: project_responsibilities.png
+    :align: center
+
+Product Sponsor (PS)
+---------------------
+
+The Project Sponsor is the person that has overall responsibility and accountability.
+
+Responsibilities
+****************
+#. Guarantee that our vision be successful by finding and securing the budget and ensuring that high-level organizational risks are mitigated.
+#. Champion the project based on whether the work fits our research needs and overall strategy.
+#. Empower the Product Owner to act for him/her on a more tactical basis.
+
+Engagement
+**********
+#. Participate in the :doc:`Events`.
+
+Product Owner (PO)
+-------------------
+
+The Product Owner is the voice of the customer and performs all tactical operations.
+
+Responsibilities
+****************
+#. Guarantee our product success by acting as a conduit/facilitator for communication between the team and the customers.
+#. Prioritizing and selecting user stories. This entails grooming the **product backlogs** with the help of the
+   Srcum Master.
+#. Break down the overall vision into feasible and tactical pieces of work for the development team to understand and create.
+#. Coordinate the :doc:`Events`.
+
+
+The means to communication to the customers include:
+
+    * Conducting Usability Tests
+    * Organizing Teaching Activities
+    * Maintaining Documentation
+    * Maintaining CEA website
+    * Maintaining cea@arch.ethz.ch
+    * Distributing Newsletters
+
+
+
+The **product backlogs** contains issues that is labelled as Must-have and Should-have.
+Here are the details of the labels used to categorize new issues in the CEA:
+
+============ =====================================================================================================
+Label        Description
+============ =====================================================================================================
+Must-have    - Features defined in the roadmap.
+             - Bugs.
+Should-have  - Issues that can't be categorized for the moment.
+Won't have   - Features that are not in the roadmap.
+             - Features that are hard to maintain. These features can become external modules of the CEA.
+============ =====================================================================================================
+
+Engagement
+**********
+#. Participate in the Review meetings (part of the :doc:`activities`).
+#. One day per week on communication and task prioritizing.
+
+
+
+Scrum Master (SM)
+-----------------
+
+The Scrum Master is the servant leader and maintainer of agile practices.
+
+Responsibilities
+****************
+#. Guarantee the success of our processes by maintain all agile practices (including :doc:`activities`).
```

Comment:
let's make this explicit, the SM (@daren-thomas ) should coordinate the activities including sprint planning, development, review and retrospective. This is based on standard practice.

PO (@shanshanhsieh ) should coordinate the roadmap meeting and any other meeting with customers, e-mail channels, website etc. 

It is better that coordination activities are distributed in this way so neither of the SM or Po is overloaded in admin, yet they can take a leading role and commit to it.